bin/niriPipe

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+import argparse
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='NIRI data processor.')
+    parser.add_argument('obsID', metavar='OBSID', type=str, nargs=1,
```

Comment:
Any reason to limit it to 1? Or better put it, could there be an advantage for processing multiple observations? Just asking.

Reply:
Very good idea, I could see that being useful for a PI of a program with many observations. Issue #12 created to keep this in mind.